include/ignition/gazebo/components/SlipComplianceCmd.hh

```diff
@@ -0,0 +1,48 @@
+
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_COMPONENTS_SLIPCOMPLIANCECMD_HH_
+#define IGNITION_GAZEBO_COMPONENTS_SLIPCOMPLIANCECMD_HH_
+
+#include <ignition/math/Vector2.hh>
+
+#include <ignition/gazebo/config.hh>
+#include <ignition/gazebo/Export.hh>
+
+#include <ignition/gazebo/components/Factory.hh>
+#include "ignition/gazebo/components/Component.hh"
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace components
+{
+  /// \brief A component type that contains the slip compliance parameters to be
+  /// set on a collision. The x and y values correspond to the slip compliance
+  /// parameters in friction direction 1 (fdir1) and friction direction 2
+  /// (fdir2) respectively.
+  using SlipComplianceCmd = Component<math::Vector2d, class SlipComplianceCmdTag>;
```

Comment:
I noticed that the `ign-physics` API is split into 2 doubles for primary and secondary compliance. I'm just pointing it out in case we want to split this too. One reason I can think of is in case we want to add more variables later.

Reply:
I agree. The user might want to set slip compliance only in one direction without affecting the other direction.

Reply:
Resolved here 139020c8ec0af219efd848b84b4954c2c65936e9